puppet/data/common.yaml

```diff
@@ -209,3 +209,7 @@ puppet::server_puppetserver_telemetry: false
 puppet::show_diff: true
 
 sudo::wheel_config: password
+
+redmine::app_root: /usr/share/redmine
+redmine::data_dir: /var/lib/redmine
```

Comment:
technically, those two are defaults, but they are overriden today for some reason, and I didn't dare to look further

Reply:
I think that's a bit of a historic thing. We can safely drop them since they do match the defaults.